.github/workflows/docsite.yml

```diff
@@ -0,0 +1,57 @@
+name: graspologic Docsite Build
+on:
+  #
+  # When a release tag is created (e.g. v1.0.0), this workflow will be triggered. The `poetry dynamic-version` plugin will use the correct version tag.
+  #
+  release:
+    types: [created]
+  #
+  #  On pushes to main and dev, a prerelease version will be cut for the branch. e.g. v1.0.0-pre.10+<hash>
+  #
+  push:
+    paths-ignore:
+      - '.all-contributorsrc'
+      - 'CONTRIBUTORS.md'
+    branches:
+      - 'main'
+      - 'dev'
+jobs:
+  publish:
+    uses: ./.github/workflows/publish.yml
+  build:
```

Comment:
what is the point of building after we're doing a build in publish.yml anyway?

Reply:
This task requires an artifact from the build job, so I _think_ this is just expressing that dependency

Reply:
The docsite is downloaded as an artifact in line 32 here, if you look at the build.yml, that's where it's created.